screens/Home/MyVcsTabMachine.ts

```diff
@@ -57,31 +53,8 @@ export const MyVcsTabMachine = model.createMachine(
       events: {} as EventFrom<typeof model>,
     },
     id: 'MyVcsTab',
-    initial: 'checkingOnboardingStatus',
+    initial: 'idle',
     states: {
-      checkingOnboardingStatus: {
```

Comment:
In the old code we were using Onboarding status variable to decide whether we need to show the intro sliders in the MyVCs screen or not and when the app is installed and download the first VC we are setting that variable to true considering that onboarding is completed but there is a issue with this approach -
 on the verifier side if we click on the received card we are creating new instance of MyVCS machine and it checks whether onboarding is completed or not since we didn't download the new vc that variable is not set to true and it will go into "Onboarding" state and inside this state there is no "VIEW_VC" event and the received card detailed view is failing because of this. 
Now we are checking whether onboarding is done or not by checking if language preference is done or not so onboarding flow is not required